RecoHGCal/TICL/interface/TICLConstants.h

```diff
@@ -0,0 +1,12 @@
+#ifndef RecoHGCal_TICL_interface_TICLConstants_h
+#define RecoHGCal_TICL_interface_TICLConstants_h
+
+#include <cstdint>
+
+namespace ticlConstants {
+constexpr float minEta = 1.5f;
+constexpr float maxEta = 3.0f;
+constexpr uint8_t maxNumberOfLayers = 52;
```

Comment:
as mentioned in numerous other PRs/issues, it is much better to get this info directly from the geometry, because it can/will change